app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsPresenter.java

```diff
@@ -37,7 +37,7 @@ public TicketsPresenter(TicketRepository ticketRepository, DatabaseChangeListene
 
     @Override
     public void start() {
-        loadTickets(false);
+        loadTickets(true);
```

Comment:
Why changing to true?

Reply:
When a ticket is created, it is inserted in the db, so ideally this should work without fetching all the tickets from server

Reply:
@iamareebjamal I checked out the code for a pretty while and found that the data being pulled on in the fragment is being accessed by the Presenter and this reload parameter has to be true is order to get refreshed. Even though the data is stored in a database, this parameter refreshes the items whenever the user returns or moves to the tickets fragment.

Reply:
true is passed when we want to fetch from internet